global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
-    "sdk":  {
-                "version":  "6.0.402",
-                "rollForward":  "feature"
-            }
-}
+    "sdk": {
+        "version": "6.0.400",
+        "rollForward": "feature"
+    }
+}
```

Comment:
Please keep the formatting of the Jay's script.

Reply:
IIRC, I was forced to update the SDK although 6.0.402 and 6.0.401 should be compatible to 6.0.400.

Reply:
Nope 401 = 6.0.9 runtime. 402 =6.0.10. Yes you can build 6.0.9 with 402 but I would keep runtime and sdk matched. So what he is doing does the work.

Reply:
Yep. I don't like the formatting but if people do use script it will prevent a big dif. I would love to adjust script to make it pretty but all the ways to do that looked hacky and I didn't like the ways. So I chose to just let script be authority of formatting.

Reply:
Since we shouldn't be changing this file during the dev cycle, and only change it on a build agent, there is no need to keep the weird formatting.